src/main/java/vectorwing/farmersdelight/common/block/WildRiceBlock.java

```diff
@@ -78,7 +79,8 @@ public void playerWillDestroy(Level worldIn, BlockPos pos, BlockState state, Pla
 		}
 
 		worldIn.levelEvent(player, 2001, pos, getId(state));
-		if (BlockTags.GUARDED_BY_PIGLINS.contains(this)) {
+		// TODO: 1.18.2 - is state == this
```

Comment:
This will need to be verified to work the same

Reply:
Thanks for pointing this out! You made me realize that the DoublePlantBlock override of this method in 1.18.2 is identical to what I copied over, so it can be entirely removed.

I can do this on my end after the merge. 👍